### PR TITLE
Bug 1168043 - [Stingray][Dashboard] Intro launch animation

### PR DESCRIPTION
**Pull request by rexboy7**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d4dcbdf1967312954c0331a613724374c5f78e35

File: tv_apps/dashboard/style/dashboard.css [hunk 1]
```diff
@@ -140,3 +140,33 @@ iframe {
   width: 100%;
   height: 100%;
 }
+
+/* Intro animation */
+.active #bottom-panel {
+  transform-origin: 50vw calc(16rem - 50vh);
```
Comment (huchengtw-moz) [inline tv_apps/dashboard/style/dashboard.css]: It's pretty hard to understand the value 16rem - 50vh. Would you mind to write more comments to explain why we use (16rem - 50vh)

Comment (rexboy7) [inline tv_apps/dashboard/style/dashboard.css]: It tries to set the origin to center of the screen, but we have to get that from point of each panel.
I'll add some comments to describe it.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2bb05e7b17846a1d0c34e602881fc6d102568fd6

**Pull request merged**